snapcraft/_store.py

```diff
@@ -348,6 +355,23 @@ def download(snap_name, channel, download_path, arch):
                 snap_name, download_path))
 
 
+def status(snap_name, series, arch):
+    store = storeapi.StoreClient()
+
+    with _requires_login():
+        status = store.get_snap_status(snap_name, series, arch)
+
+    parsed_channels = [
+        channel
+        for arch, channel_map in sorted(status.items())
+        for channel in _format_channel_map(channel_map, arch)]
+
+    tabulated_channels = tabulate(
+        parsed_channels, headers=['Arch', 'Channel', 'Version', 'Revision'],
+        tablefmt="plain")
```

Comment:
oh, we have in a guy in our team that told us to always use single quotes unless it is a docstring ;-)